.github/workflows/desktop-e2e-podman-windows-matrix.yaml

```diff
@@ -115,7 +115,9 @@ jobs:
             --windows-featurepack '${{ matrix.windows-featurepack }}' \
             --vmsize 'Standard_D8s_v4' \
             --tags project=podman-desktop \
-            --spot
+            --spot \
+            # Workaround for https://github.com/redhat-developer/mapt/issues/451
```

Comment:
you cannot use comment in multiline command. It will break next line as it will be treated as a another command to call.